README.md

```diff
@@ -14,55 +14,59 @@ We will know that we are progressing forward on this problem if people find the
 
 ### Our vision is that anyone can understand the impact of impending weather. When it comes to making decisions to save life and property, every word and every minute matters.
 
-**Our mission** is to rebuild weather.gov to reflect the integrity and care NWS has for the people you serve. 
```

Comment:
My prettier config cleans up Markdown files. I highly recommend reviewing this file with the rich diff instead of the source diff.